test/testcafe/framework/page-objects/DuoPageObject.js

```diff
@@ -1,14 +1,25 @@
 import BasePageObject from './BasePageObject';
+import { ClientFunction } from 'testcafe';
 
 export default class DuoPageObject extends BasePageObject {
   constructor(t) {
     super(t);
   }
 
   async clickDuoMockLink() {
-    await this.t.switchToIframe('iframe');
```

Comment:
TestCafe 3 produces error
```
Content of the iframe to which you are switching did not load.
```
(even with timeouts)

Reply:
I'm curious if we even still need to support Duo (the legacy iframe way). I recall it was deprecated by Duo or will be soon.

Reply:
> Support for the traditional Duo Prompt experience and Duo Prompt delivery via iframe ended on March 30, 2024 for most applications.

https://duo.com/docs/universal-prompt-update-guide